CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Unreleased
 
 ### Removed
 * Removed DangerJS integration
+* Removed Terra prefix from mixin pattern rule.
```

Comment:
This is probably being nit picky, but I think this belongs in the Changed section. We didn't remove any functionality, we changed the existing functionality.

Reply:
Resolved in 7346f40